smallrye-reactive-messaging-kafka/src/main/java/io/smallrye/reactive/messaging/kafka/impl/KafkaClientServiceImpl.java

```diff
@@ -25,7 +25,7 @@
     }
 
     @Override
-    public <K, V> Producer<K, V> getProducer(String channel) {
+    public <K, V> KafkaProducer<K, V> getProducer(String channel) {
```

Comment:
That is a breaking change, so we need to remember to mention it to the migration guide (in Quarkus)

Reply:
+1